modules/ztp-applying-the-ran-policies-for-monitoring-cluster-activity.adoc

```diff
@@ -0,0 +1,22 @@
+/ Module included in the following assemblies:
```

Comment:
/ Module included in the following assemblies: is appearing in the preview, FYI.

Reply:
Changed.